### PR TITLE
feat: changes for version 2.0.0

### PR DESCRIPTION
**Pull request by lleifermann**

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h2 id="subtitle">v1.0.0</h2>
       var respecConfig = {
         specStatus: 'base',
         shortName: 'Credential Issuer Conformance Criteria',
-        publishDate: '4 Feb 2022',
```
Comment (britpayson) [inline content/index.html]: Keep published date as '4 Feb 2022' for v2.0.0

File: content/index.html [hunk 2]
```diff
@@ -231,9 +271,9 @@ <h2>Regulatory Compliance – Federal Mandates</h2>
       <section>
         <h2>Digital Ecosystems - Identity of Transacting Entities*</h2>
```
Comment (britpayson) [inline content/index.html]: Remove the asterisk at the end of 'Entities'

File: content/index.html [hunk 3]
```diff
@@ -231,9 +271,9 @@ <h2>Regulatory Compliance – Federal Mandates</h2>
...
         <p>
-          Verifying that a company is “authorized” is enough to meet the letter
+          Verifying that an organization is “authorized” is enough to meet the letter
           of the law; however, in digital ecosystems it is also necessary to
-          establish the <i>identity</i> of the company. Thus, it becomes a two part
+          establish the <i>identity</i> of the organization. Thus, it becomes a two part
           process – is the counterparty who they say they are, and if so, are
           they “authorized”? To add to this challenge, DSCSA transactions for PI
           Verification and TI Tracing move through any number of intermediaries
```
Comment (britpayson) [inline content/index.html:275]: Remove the line about "*Content remains under discussion. OCI expects to resolve feedback by Version 2.0 release in December 2021."

File: content/index.html [hunk 4]
```diff
@@ -251,19 +291,31 @@ <h2>Digital Ecosystems - Identity of Transacting Entities*</h2>
...
+          In the future, the Root of Trust could be instantiated on a higher level by establishing a body that certifies the
+          Credential Issuers, and then issues verifiable credentials to them. As there are very few Credential Issuers in the
+          initial phase of OCI, such a process is not necessary. OCI uses the DID:web method for the Credential Issuers as this is
+          a method built upon proven, well-known instruments. The initial OCI implementation includes a registry with the names
+          and DIDs of vetted Credential Issuers that have demonstrated and proven adherence with the OCI Credential Issuer
+          Conformance Criteria.
+        </p>
       </section>
       <section>
         <h2>Purpose of Conformance Criteria*</h2>
```
Comment (britpayson) [inline content/index.html]: Remove the asterisk after 'Criteria' and remove the statement about "*Content remains under discussion. OCI expects to resolve feedback by Version 2.0 release in December 2021."

File: content/index.html [hunk 5]
```diff
@@ -449,6 +501,65 @@ <h2>W3C Standards for Credential Formatting</h2>
           </a>
```
Comment (britpayson) [inline content/index.html]: Prior to (line 429 -  The World Wide Web Consortium (W3C) is an international community), we need to add the section title <h2>About (W3W)</h2>

File: content/index.html [hunk 6]
```diff
@@ -565,6 +676,17 @@ <h2>
               if the user consents.
```
Comment (britpayson) [inline content/index.html:667]: Update line 514 to Identity Assurance Levels (IAL)</a>

File: content/index.html [hunk 7]
```diff
@@ -752,7 +872,32 @@ <h2>Evidence Quality – Superior, Strong, and Fair</h2>
             </i>
```
Comment (britpayson) [inline content/index.html]: Delete all of the follow from the list above:
•	The issuing source visually identified the applicant and performed further checks to confirm the existence of that person. 
•	The issuing process for the evidence ensured that it was delivered into the possession of the person to whom it relates. 
•	The evidence contains at least one reference number that uniquely identifies the person to whom it relates. 
•	The full name on the evidence must be the name that the person was officially known by at the time of issuance. Not permitted are pseudonyms, aliases, an initial for surname, or initials for all given names. 
•	The evidence contains a photograph of the person to whom it relates. 
•	The evidence includes physical security features that require proprietary knowledge and proprietary technologies to be able to reproduce it. 
•	The evidence is unexpired.

File: content/index.html [hunk 8]
```diff
@@ -768,16 +913,15 @@ <h2>Evidence Quality – Superior, Strong, and Fair</h2>
         </p>
       </section>
       <section>
-        <h2>Use of DEA Signing Certificate as Superior Evidence</h2>
+        <h2>Examples of the Use of One Piece of Strong Evidence</h2>
```
Comment (britpayson) [inline content/index.html:864]: Below this there should be a subtitle "4.2.1 DEA Signing Certificate"

File: content/index.html [hunk 9]
```diff
@@ -826,18 +970,32 @@ <h2>Use of DEA Signing Certificate as Superior Evidence</h2>
             </i>
           </li>
         </ul>
+        <section>
+          <h2>National Provider Identifier</h2>
```
Comment (britpayson) [inline content/index.html:925]: Update to read as "National Provider Identifier (NPI)

File: content/index.html [hunk 10]
```diff
@@ -958,8 +1117,8 @@ <h2>IRS Employer ID Number (EIN) Letter</h2>
             The original EIN letter (form CP 575) was mailed by the IRS to the
             address specified on the application form SS-4, and is not
             downloadable from a website. Therefore, it would be quite difficult
-            for someone not associated with the company to get a copy of this
-            document. A company can get a replacement (147c letter) by knowing
+            for someone not associated with the organization to get a copy of this
+            document. A organization can get a replacement (147c letter) by knowing
```
Comment (britpayson) [inline content/index.html]: Change from "A organization" to "An organization"

**Review by britpayson — CHANGES_REQUESTED** [review]
@lleifermann all of the edits made are approved. I added comments for specifics on what still needs to be changed.

File: content/index.html (hunk 6 above)
Comment (lleifermann) [reply content/index.html:667]: If you intend to show that  Identity Assurance Levels is an abbrevation. There is a place for that in the Terms and Abbreviations section. Adding this here will lead to an Error as you are specifing an abbrevation outside of that place.

Comment (britpayson) [reply content/index.html:667]: Understood.

File: content/index.html (hunk 9 above)
Comment (lleifermann) [reply content/index.html:925]: If you intend to show that National Provider Identifier is an abbrevation. There is a place for that in the Terms and Abbreviations section. Adding this here will lead to an Error as you are specifing an abbrevation outside of that place.

Comment (britpayson) [reply content/index.html:925]: Understood.

File: content/index.html (hunk 8 above)
Comment (lleifermann) [reply content/index.html:864]: Fixed as well!

File: content/index.html (hunk 4 above)
Comment (britpayson) [reply content/index.html]: Need the asterisk removed after Criteria in the subtitle.

**Review by britpayson — CHANGES_REQUESTED** [review]
@lleifermann just a couple of edits still pending so I re-opened those related comments.

**Review by britpayson — APPROVED** [review]

**Pull request merged**